lib/class-controller.php

```diff
@@ -242,6 +242,15 @@ public function enable_relative_date_aggregation( array $args = [] ): Controller
 		return $this;
 	}
 
+	/**
+	 * Adds a custom Aggregation to the list of active aggregations.
+	 *
+	 * @param Aggregation $aggregation The aggregation to add.
+	 */
+	public function add_custom_aggregation( Aggregation $aggregation ): void {
```

Comment:
Name this `add_aggregation` and have it pass through to the (now public, based on previous comment) `add_aggregation` method.

Reply:
This still needs to be done.